src/handlers/issue-deduplication.ts

```diff
@@ -69,42 +86,46 @@ async function handleSimilarIssuesComment(context: Context, payload: IssuePayloa
             ... on Issue {
               title
               url
+              repository {
+                name
+                owner {
+                  login
+                }
+              }
             }
           }
         }`,
         { issueNodeId: issue.issue_id }
       );
-      issueUrl.similarity = (issue.similarity * 100).toFixed(2);
+      issueUrl.similarity = Math.round(issue.similarity * 100).toString();
       return issueUrl;
     })
   );
 
-  const commentBody = issueList.map((issue) => `- [${issue.node.title}](${issue.node.url}) Similarity: ${issue.similarity}`).join("\n");
-  const body = `This issue seems to be similar to the following issue(s):\n\n${commentBody}`;
+  let finalIdx = 0;
```

Comment:
Never shorten words in identifiers to reduce cognitive overhead. I presume this is meant to mean `finalIndex`

Reply:
```suggestion
  let finalIndex = 0;
```